crates/bevy_ui/src/entity.rs

```diff
@@ -21,8 +25,22 @@ use bevy_transform::prelude::{GlobalTransform, Transform};
 pub struct NodeBundle {
     /// Describes the size of the node
     pub node: Node,
-    /// Describes the style including flexbox settings
-    pub style: Style,
+    /// The layout algorithm used
+    pub layout_strategy: LayoutStrategy,
+    /// The offset of this UI node, relative to its default position
+    pub offset: Offset,
+    /// Whether the node should be absolute or relatively positioned
+    pub position_type: PositionType,
+    /// The constraints on the size of this node
+    pub size_constraints: SizeConstraints,
+    /// The margin, padding and border of the UI node
+    pub spacing: Spacing,
+    /// The flexbox layout parameters
+    pub flex_layout: FlexLayout,
```

Comment:
I'm not too sure having both `flex_layout` and `layout_strategy` together would be the best option. `Taffy` does have plans to support layouts other than flex (e.g. grid), so I would suggest a more generic abstraction such as `layout` here for future concerns.

Reply:
Perhaps an even better idea here is to remove `layout_strategy` and replace `flex_layout` with `layout` as a enum which takes `AbsoluteLayout` and `FlexLayout`?

Reply:
I like that latter idea a lot.